.kokoro/downstream-client-library-check.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eo pipefail
+# Display commands being run.
+set -x
+
+
+CORE_LIBRARY_ARTIFACT=$1
+CLIENT_LIBRARY=$2
+## Get the directory of the build script
+scriptDir=$(realpath $(dirname "${BASH_SOURCE[0]}"))
+
+mkdir /tmp/foo && cd /tmp/foo
+wget https://dl.google.com/dl/android/maven2/com/google/android/gms/play-services-basement/8.3.0/play-services-basement-8.3.0.aar
+unzip play-services-basement-8.3.0.aar
+mvn install:install-file \
+  -Dfile=classes.jar \
+  -DgroupId=com.google.android.google-play-services \
+  -DartifactId=google-play-services \
+  -Dversion=1 \
+  -Dpackaging=jar
```

Comment:
You might wish to explain why you need this.

Reply:
Done